bucket_store.gemspec

```diff
@@ -16,16 +16,18 @@ Gem::Specification.new do |s|
 
   s.files = Dir["lib/**/*", "README.md"]
 
-  s.required_ruby_version = ">= 2.6"
+  s.required_ruby_version = ">= 2.7"
```

Comment:
Wouldn't this be a breaking change? Should this be a major version bump? Also are we planning to do a 1.0 release?